chapters/connectors.tex

```diff
@@ -903,7 +905,31 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
-Before connection equations are generated, the virtual connection graph is transformed into a set of spanning trees by removing optional spanning tree edges from the graph.
+When generating connection graph equations, only the overdetermined components of a connector are considered.
+The connection graph equations replace the equality-equations for variables that are neither flow nor stream in \cref{generation-of-connection-equations}.
+
+\subsubsection{Remove Redundant Connect-Equations}
+In order to eliminate any redundant \lstinline!connect!-equation the following preparation is needed.
+\begin{nonnormative}
+In the common case where there are no redundant \lstinline!connect!-equations a \lstinline!connect! between connectors with \emph{one} overdetermined component will directly generate \emph{one} breakable branch.
+\end{nonnormative}
+\begin{enumerate}
+\item
+  The connection sets are built similarly to the normal way, but keeping the overdetermined components as primitives.
+\item
+  Instead of generating the equality-equation for an overdetermined component, a breakable branch in the virtual connection graph is constructed.
+\end{enumerate}
+If a connection set contains $n$ overdetermined components, and was built from $m$ \lstinline!connect!-equations, then $m-n-1$ of the \lstinline!connect!-equations are redundant.
+If there are no redundant \lstinline!connect!-equations (i.e., if $m=n-1$), the breakable branches can be chosen to correspond to the \lstinline!connect!-equations for overdetermined connectors.
+If there are redundant \lstinline!connect!-equations there will always exist \lstinline!connect!-equations without a corresponding breakable branch.
+\begin{nonnormative}
+They are called redundant since if they were removed the same connection set and equations would be generated.
+Redundant \lstinline!connect!-equations include directly connecting a connector to itself, duplicated connections, and one connection among all of the pair-wise connections between \lstinline!a!, \lstinline!b!, and \lstinline!c!.
+The latter case can be used to consistently handle conditional components (so that disabling \lstinline!b! does not break the connection between \lstinline!a! and \lstinline!c!).
+\end{nonnormative}
+
+\subsubsection{Actual Generation of Connection Equations}
```

Comment:
It would be good to not have a subsubsection named like this.  How about naming the two subsubsection `\subsubsection{Spanning Trees}` and `\subsection{Equations}`, and move the subsubsection split so that the first ends with:
> The virtual connection graph is finally transformed into a set of spanning trees by removing optional spanning tree edges from the graph.

Reply:
I couldn't find that exact text, and found it clearer to split the subsubsection into two.